app/Components/RegisterUser/RegisterUser.js

```diff
@@ -219,7 +216,7 @@ const styles = StyleSheet.create({
         color:"red"
     }
 });
-const mapStateToProps = (state) => (console.log("console in msp", state), {
+const mapStateToProps = (state) => ( {
     details: state,
```

Comment:
need specific naming conventions here there is no meaning for details in this section.